icevision/models/fastai/unet/fastai/callbacks.py

```diff
@@ -10,3 +11,15 @@ def before_batch(self):
         self.learn.records = self.yb[0]
         self.learn.yb = (self.xb[0][1],)
         self.learn.xb = (self.xb[0][0],)
+
+    def after_pred(self):
```

Comment:
Why was this one added?
Was there something broken with the UNet implementation?

Reply:
Indeed - the original `unet` implementation did not convert predictions to the "standard" IceVision format and was carrying out operations using the raw output format. I think this was fine before as it was kind of a stand-out model being the only way to do semantic segmentation. However, it meant that it was not compatible with everything else being built for semantic seg (in this PR and elsewhere).

For example, it was not possible to have both `unet` and `mmsegmentation` models in the same tutorial notebook as `unet` would fail with the newly introduced semantic segmentation metrics.